helm/dendrite/templates/secrets.yaml

```diff
@@ -30,88 +30,63 @@ apiVersion: v1
-      server_name: {{ .Values.configuration.server_name }}
+      server_name: {{ .Values.dendrite_config.global.server_name }}
       private_key: /etc/dendrite/secrets/signing.key
       key_id: ed25519:auto
-      key_validity_period: {{ .Values.configuration.key_validity_period | quote }}
+      key_validity_period: {{ .Values.dendrite_config.global.key_validity_period | quote }}
       database:
         connection_string: {{ $connectionString }}?sslmode=disable
-        max_open_conns: {{ default 90 .Values.configuration.database.max_open_conns }}
-        max_idle_conns: {{ default 5 .Values.configuration.database.max_idle_conns }}
-        conn_max_lifetime: {{ default -1 .Values.configuration.database.conn_max_lifetime }}
+        max_open_conns: {{ .Values.dendrite_config.global.database.max_open_conns }}
+        max_idle_conns: {{ .Values.dendrite_config.global.database.max_idle_conns }}
+        conn_max_lifetime: {{ .Values.dendrite_config.global.database.conn_max_lifetime }}
       cache:
-        max_size_estimated: {{ default "1gb" .Values.configuration.cache.max_size_estimated | quote }}
-        max_age: {{ default "1h" .Values.configuration.cache.max_age }}
-      well_known_server_name: {{ default "" .Values.configuration.well_known_server_name | quote }}
-      well_known_client_name: {{ default "" .Values.configuration.well_known_client_name | quote }}
+        max_size_estimated: {{ .Values.dendrite_config.global.cache.max_size_estimated | quote }}
+        max_age: {{ .Values.dendrite_config.global.cache.max_age }}
+      well_known_server_name: {{ .Values.dendrite_config.global.well_known_server_name | quote }}
+      well_known_client_name: {{ .Values.dendrite_config.global.well_known_client_name | quote }}
       trusted_third_party_id_servers:
-      {{- toYaml .Values.configuration.trusted_third_party_id_servers | nindent 8 }}
-      disable_federation: {{ .Values.configuration.disable_federation }}
+      {{- toYaml .Values.dendrite_config.global.trusted_third_party_id_servers | nindent 8 }}
+      disable_federation: {{ .Values.dendrite_config.global.disable_federation }}
+      report_stats:
+        {{- toYaml .Values.dendrite_config.global.report_stats | nindent 8 }}
+      presence:
+        {{- toYaml .Values.dendrite_config.global.presence | nindent 8 }}
+      server_notices:
+        {{- toYaml .Values.dendrite_config.global.server_notices | nindent 8 }}
       jetstream:
         in_memory: false
         storage_path: /data/jetstream
       metrics:
-        {{- toYaml .Values.configuration.metrics | nindent 8 }}
+        {{- toYaml .Values.dendrite_config.global.metrics | nindent 8 }}
       dns_cache:
-        {{- toYaml .Values.configuration.dns_cache | nindent 8 }}
+        {{- toYaml .Values.dendrite_config.global.dns_cache | nindent 8 }}
     mscs:
       mscs:
-        {{- toYaml .Values.configuration.mscs | nindent 8 }}
+        {{- toYaml .Values.dendrite_config.mscs.mscs | nindent 8 }}
```

Comment:
As discussed yesterday, please can we just drop-in `.Values.dendrite_config` directly into the value for this `stringData`? That way we don't need to update this file when config options change, and it forces us to put all defaullts in `Values.yml`.